lib/foreman_ansible_core/ansible_runner_task_launcher.rb

```diff
@@ -43,13 +49,15 @@ def handle_event_file(event_file)
             handle_broadcast_data(event)
           end
           true
-        rescue JSON::ParserError
-          nil
+        rescue JSON::ParserError => e
+          logger.error("[foreman_ansible] - Error parsing runner event at #{event_file}: #{e.class}: #{e.message}")
+          logger.debug(e.backtrace.join("\n"))
         end
       end
 
       def handle_host_event(hostname, event)
-        publish_data_for(hostname, event['stdout'], 'stdout')
+        log_event("for host: #{hostname.inspect}", event)
+        publish_data_for(hostname, event['stdout'], 'stdout') if event['stdout']
```

Comment:
I've notice the output between `ansible-playbook` and `ansible-runner` differed: the runner was missing the output data from the task. It looks much nicer (colors and everything) when publishing this data as well + it's quite helpful.